webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue

```diff
@@ -27,6 +27,10 @@
                 class="ml-2 ignore-vuetify-classes activityShareDescription">
               </textarea>
             </div>
+            <div class="d-flex flex-row mt-4 ml-2">
+              <v-icon size="20px" color="#f8b441">warning</v-icon>
```

Comment:
- color and size should not be added directly, use css helpers instead
- the warning message should be added only if the description contains a link

Reply:
-done